processing/src/feat_comp.cc

```diff
@@ -209,3 +238,27 @@ void FeatComp::_add_jet_flags(const float& b_1_csv, const float& b_2_csv,
     if (FeatComp::_feat_check("jet_1_quality")) feats["jet_1_quality"] = tag_1;
     if (FeatComp::_feat_check("jet_2_quality")) feats["jet_2_quality"] = tag_2;
 }
+
+inline float FeatComp::calc_centrality( const LorentzVector& v, const LorentzVector& VBFjet1, const LorentzVector& VBFjet2)
+{
+    return (v.Eta() - 0.5*(VBFjet1.Eta() + VBFjet2.Eta() ))/(std::fabs(VBFjet1.Eta() - VBFjet2.Eta()));
+}
+
+inline float FeatComp::calc_hh_centrality( const LorentzVector& bh, const LorentzVector& tauh, const LorentzVector& VBFjet1, const LorentzVector& VBFjet2)
+{
+    return  FeatComp::calcDeltaEtaMinus(bh, tauh, VBFjet1, VBFjet2) <  FeatComp::calcDeltaEtaPlus(bh, tauh, VBFjet1, VBFjet2) ?  FeatComp::calcDeltaEtaMinus(bh, tauh, VBFjet1, VBFjet2) :  FeatComp::calcDeltaEtaPlus(bh, tauh, VBFjet1, VBFjet2);
```

Comment:
Again, can you cache the delta etas to save having to compute them twice

Reply:
Done. Also modified the definition of both calcDeltaEtaMinus and Plus to cache all etas.